### PR TITLE
Splitting shipambience.ogg from toggle ambience

### PR DESCRIPTION
**Pull request by Bram961**
Splits off Ambience sounds and the background "shipambience.ogg" under
two difrent buttons.

### CODE REVIEW

File: code/modules/client/preferences_toggles.dm [hunk 1]
```diff
@@ -131,9 +131,22 @@
 	else
 		src << "You will no longer hear ambient sounds."
 		src << sound(null, repeat = 0, wait = 0, volume = 0, channel = 1)
-		src << sound(null, repeat = 0, wait = 0, volume = 0, channel = 2)
```
Comment (Fox-McCloud) [inline code/modules/client/preferences_toggles.dm]: uhhhhh, won't this just kill it on a single channel?

File: code/modules/client/preferences_toggles.dm [hunk 2]
```diff
@@ -131,9 +131,22 @@
...
+	set name = "Hear/Silence White Noise"
+	set category = "Preferences"
+	set desc = "Toggles hearing ambient white noise"
+	prefs.sound ^= SOUND_BUZZ
+	prefs.save_preferences(src)
+	if(prefs.sound & SOUND_BUZZ)
+		src << "You will now hear ambient white noise."
+	else
+		src << "You will no longer hear ambient white noise."
+		src << sound(null, repeat = 0, wait = 0, volume = 0, channel = 2)
```
Comment (Fox-McCloud) [inline code/modules/client/preferences_toggles.dm]: see above

**Comment by Bram961** [comment]
No because the shipambience.ogg plays over channel 2 http://gyazo.com/14f2c35b0ad71d3c14c8960254c66c4a 
and the other ambience plays over channel 1 http://gyazo.com/86a55ad4b4ca457e50f5753b84f97057
So where before the button that toggles all of that would turn both of, I've split that as well.

**Comment by Bram961** [comment]
this was tested as well, Doing it this way allows you to turn off ambience withouth turning off the background noise and vice versa. doing it with both channels causes shipambience.ogg to do weird things either turning it off temporarily or turning it off permanently when the other ambient noises are toggled (I did this at 3 o clock last night and im not quite sure which issue it was but I split the channels for a reason)

**Comment by ZomgPonies** [comment]
Well, I'll need to overhaul the whole preferences menu after this and put it into a fancy karma-like menu with tabs showing which preferences are active. The current preferences tab is getting pretty convoluted.

**Pull request merged**